docker-compose.dev.yaml

```diff
@@ -1,42 +1,15 @@
 version: "3.7"
 
 services:
+  # hook in case we need to add init behavior
+  # every root service (no depends_on) should depend on init
   init:
-    command: /bin/sh -c "
-      ./scripts/create_mount_directories.sh /local_parent ${HACK_LOCAL_ROOT_PARENT} ${LOCAL_ROOT};
-      ./scripts/create_mount_directories.sh /dev_parent ${HACK_DEV_ROOT_PARENT} ${DEV_ROOT}/db;
-      ./scripts/create_mount_directories.sh /dev_parent ${HACK_DEV_ROOT_PARENT} ${DEV_ROOT}/data;
-      ./scripts/create_mount_directories.sh /dev_parent ${HACK_DEV_ROOT_PARENT} ${DEV_ROOT}/workspace;
-      "
-    environment:
-      - HACK_DEV_ROOT_PARENT=${HACK_DEV_ROOT_PARENT}
-      - DEV_ROOT=${DEV_ROOT}
-      - HACK_LOCAL_ROOT_PARENT=${HACK_LOCAL_ROOT_PARENT}
-      - LOCAL_ROOT=${LOCAL_ROOT}
-    volumes:
-      - ${HACK_DEV_ROOT_PARENT}:/dev_parent
-      - ${HACK_LOCAL_ROOT_PARENT}:/local_parent
-  db:
-    ports:
-      - 5432:5432
-    volumes:
-      - ${DEV_ROOT}/db:/var/lib/postgresql/data
+    container_name: init-dev
```

Comment:
uses exactly docker-compose.yaml except for overriding container_name. names different to avoid collision.